### DISCUSSION TITLE
why this rune transfer without edicts?

### DISCUSSION THREAD

**Discussion by Ljzn** [ordinals/ord#3859] [Q&A] [answered]
Hello, I'm learning the runes protocol. The specification says runes be transferred with edicts, but I found this tx:

https://ordiscan.com/tx/a541cf35eed3f17651f5a44ef5a9a41ac23ad465ee9ef2ef865691fc5915ed4c

which transfers runes without any op_return output. Is this a hidden rule of runes?

**Comment by raphjaph** [comment, marked as answer]
It's not made explicit in the docs but if you read these two parts from https://docs.ordinals.com/runes.html, you can see that a tx with no edict but runes in any of the inputs, transfers them to the first non-`OP_RETURN` output.


>[Edicts](https://docs.ordinals.com/runes.html#edicts)
>A runestone may contain any number of edicts. Edicts consist of a rune ID, an amount, and an output number. Edicts are processed >in order, allocating unallocated runes to outputs.
>
>[Pointer](https://docs.ordinals.com/runes.html#pointer)
>After all edicts are processed, remaining unallocated runes are transferred to the transaction's first non-OP_RETURN output. A runestone may optionally contain a pointer that specifies an alternative default output.

**Reply by Ljzn** [reply]
Thank you!
